src/ansible_navigator/initialization.py

```diff
@@ -23,6 +24,11 @@
 from .utils import find_settings_file
 
 
+if TYPE_CHECKING:
+
+    from ...share.ansible_navigator.utils import key_value_store  # type: ignore[misc]
```

Comment:
This import, and the required ignore for it, seem odd to me. Time to move `key_value_store` into `src/ansible_navigator`? This would let us kill the (also weird to me) `_get_kvs()` below, too.

Reply:
Maybe? Let's talk more about this...........  It just feels weird to me if we were pulled navigator internals into into the EE to introspect it. Maybe that's not an issue.  We could maybe talk about a new parameter or flag to do the introspection and maybe pick that off early in cli.py?

BTW, when I did that TYPING ...  I was thinking about the fact it was odd and there should be a better way.

Reply:
@relrod ping so we remember to talk more about this.